### PR TITLE
Enable customer to customize the incoming account ident

### PR DESCRIPTION
**Pull request by michaloo**

### CODE REVIEW

**Comment by unity** [comment]
Can you explain the « why »?

**Comment by michaloo** [comment]
@unity sure, I'm giving more info in this ticket: https://github.com/hull/hull-connectors-issues/issues/115

File: server/lib/sync-agent/mapping-util.js [hunk 1]
```diff
@@ -405,17 +407,34 @@ class MappingUtil {
...
+      hubspotCompany.properties.domain.value;
+
+    if (
+      domainIdentity !== undefined &&
+      domainIdentity !== null &&
+      domainIdentity.trim() !== ""
+    ) {
+      ident.domain = domainIdentity;
+    }
+
```
[the remark this answers is not in the document]
Comment (michaloo) [reply server/lib/sync-agent/mapping-util.js]: @timliuhull yes, the behavior is explained here in the settings: https://github.com/hull-ships/hull-hubspot/pull/95/files#diff-4b1eb3dc48c4e16d49db5b42298fe654R146

File: manifest.json [hunk 2]
```diff
@@ -85,6 +85,13 @@
         }
       }
     },
+    {
+      "name": "outgoing_account_title",
+      "title": "Outgoing accounts configuration",
+      "description": "Outgoing accounts are matched on Hubspot end by `account.domain`. Accounts without domain are skipped. If you want to send out `account.external_id` add it to the table below.",
```
Comment (unity) [inline manifest.json]: Let's not create a new strategy and UX here. I haven't seen any other example of us exposing a free-form string.

Comment (michaloo) [reply manifest.json]: @unity free form string? This json entry you are commenting on is just a title.

Comment (unity) [reply manifest.json]: Sorry, my mistake

**Comment by michaloo** [comment]
Accepted to push this behavior for accounts instance.

**Pull request merged**